app/service/cloud_watch_service.rb

```diff
@@ -0,0 +1,37 @@
+class CloudWatchService
+  def self.week_submissions(form_id:)
+    region = "eu-west-2"
+    a_week = 604_800
```

Comment:
Does it really matter? As long as it still returns a good enough answer, it'll be fine.

Reply:
The accuracy doesn't bother me too much as long as we're confident CloudWatch won't do anything unexpected in this case (I don't really know what the CloudWatch response looks like or if it's different when the `period` isn't the same as the time between the start and end dates)

Reply:
I suppose it's fairly low risk since:
- we're building this feature to handle cloudwatch errors gracefully
- the clocks go back later this month so we'll find out soon enough

Reply:
Just tested this, a period that overruns the start of the query isn't an issue - datapoints that aren't within the start and end times just aren't included in the data.

Reply:
And yeah for the scenario where the period doesn't cover the first hour of the saved day, we'll miss stats from that hour as it'll be part of another datapoint which we're not using. 

We could fix this by checking the length of the diff between the two points in time and seeing if they match the period. I'm leaning towards not bothering right now!

Reply:
That sounds alright to me - thanks for checking!

Reply:
Yay! Nice one! Thanks for checking Sam